template-parts/impact/browser/header.php

```diff
@@ -8,6 +8,7 @@
 $header_img             = $options->header_image ?? null;
 $header_img_utm_content = get_option( 'impact_header_utm_content' ) ?: '';
 $header_img_url         = isset( $options->header_image_link ) ? Impact\format_url_utm_params( $options->header_image_link, $header_img_utm_content ) : null;
+$header_image_alt       = $options->header_image_alt ?? '';
```

Comment:
Consider passing `$options->header_image_alt` through `esc_attr()` to prevent double quotes or weird characters from breaking the email markup: https://developer.wordpress.org/reference/functions/esc_attr/